genai-perf/genai_perf/inputs/inputs_config.py

```diff
@@ -53,89 +53,112 @@
 class InputsConfig:
     """
     A class to hold all of the arguments used for creating the inputs
+    """
 
-      Required Parameters
-      -------------------
-      input_type:
-          Specify how the input is received
-      output_format:
-          Specify the output format
-
-      Optional Parameters
-      -------------------
-      dataset_name:
-          The name of the dataset
-      model_name:
-          The model name
-      starting_index:
-          Offset from within the list to start gathering inputs
-      length:
-          Number of entries to gather
-      add_stream:
-          If true, adds a steam field to each payload
-      extra_inputs:
-          If provided, append these inputs to every request
-      output_tokens_mean:
-          The mean length of the output to generate. If not using fixed output lengths, this should be set to -1.
-      output_tokens_stddev:
-          The standard deviation of the length of the output to generate. This is only used if output_tokens_mean is provided.
-      output_tokens_deterministic:
-          If true, the output tokens will set the minimum and maximum tokens to be equivalent.
-      image_width_mean:
-          The mean width of images when generating synthetic image data.
-      image_width_stddev:
-          The standard deviation of width of images when generating synthetic image data.
-      image_height_mean:
-          The mean height of images when generating synthetic image data.
-      image_height_stddev:
-          The standard deviation of height of images when generating synthetic image data.
-      image_format:
-          The compression format of the images.
-      batch_size:
-          The number of inputs per request (currently only used for the embeddings, image retrieval, and rankings endpoints)
-
-      Required Synthetic Prompt Generation Parameters
-      -----------------------------------------------
-      tokenizer:
-         The tokenizer to use when generating synthetic prompts
-
-      Optional Synthetic Prompt Generation Parameters
-      -----------------------------------------------
-      prompt_tokens_mean:
-          The mean length of the prompt to generate
-      prompt_tokens_stddev:
-          The standard deviation of the length of the prompt to generate
-      num_prompts:
-          The number of synthetic output prompts to generate
-      random_seed:
-          Seed used to generate random values
+    ####################
+    # General Parameters
+    ####################
 
-    """
+    # add_stream:
```

Comment:
This is definitely an improvement! But, it seems like listing the variable name above the comment is redundant and could be removed while keeping the same level of readability.

Reply:
You are totally right. 
Got rid of the redundancy.